script/slurm/submit_slurm_jobs.sh

```
@@ -292,11 +298,19 @@ if [[ "${PIPELINE_MODE}" == "two_stage" ]]; then
       ${NSS_SLURM_DIR}/slurm_srun.sh )
 
 else
-  sbatch "${common_args[@]}" \
+  end_to_end_array_id=$( \
+    sbatch "${common_args[@]}" \
     --time="${TIME_LIMIT}" \
     --job-name nss_end_to_end \
     --export=ALL,NSS_PHASE=end_to_end \
-    ${NSS_SLURM_DIR}/slurm_srun.sh
+    ${NSS_SLURM_DIR}/slurm_srun.sh )
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)